pkgs/applications/radio/sdrangel/default.nix

```diff
@@ -0,0 +1,73 @@
+{
+boost,
+cmake,
+cm256cc,
+fetchFromGitHub,
+fftwFloat,
+glew,
+libav,
+libiio,
+libopus,
+libpulseaudio,
+libusb,
+limesuite,
+mkDerivation,
+ocl-icd,
+opencv3,
+pkgconfig,
+pkgs,
+qtbase,
+qtmultimedia,
+qtwebsockets,
+serialdv,
+stdenv,
+wrapQtAppsHook,
+}:
+
+let
+
+  codec2 = pkgs.codec2.overrideAttrs (old: {
+    src = fetchFromGitHub {
+      owner = "drowe67";
+      repo = "codec2";
+      rev = "567346818c0d4d697773cf66d925fdb031e15668";
+      sha256 = "0ngqlh2cw5grx2lg7xj8baz6p55gfhq4caggxkb4pxlg817pwbpa";
+    };
+  });
+
+in mkDerivation rec {
+  pname = "sdrangel";
+  version = "4.11.7";
+  src = fetchFromGitHub {
+    owner = "f4exb";
+    repo = "sdrangel";
+    rev = "v${version}";
+    sha256 = "0zbx0gklylk8npb3wnnmqpam0pdxl40f20i3wzwwh4gqrppxywzx";
+    fetchSubmodules = false;
+  };
+  nativeBuildInputs = [ cmake pkgconfig ];
+  buildInputs = [
+    glew opencv3 libusb boost libopus limesuite libav libiio libpulseaudio
+    qtbase qtwebsockets qtmultimedia wrapQtAppsHook
```

Comment:
I think `wrapQtAppsHook` should be in `nativeBuildInputs`

Reply:
No it shouldn't read my reply in the one where you suggested replacing mkDerivation to stdenv.mkDerivation ^

Reply:
The manual states that you should use `wrapQtAppsHook` only when you can't use the qt provided `mkDerivation` (which you are using here). This seems to be redundant?

Reply:
yeah i just realised i left it here, removed it just now